.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+# Template meant to be filled out when filing new issues
```

Comment:
Includes additional changes. Include in only yours.

Reply:
@intkhabahmed Add additional changes, I need ur changes to proceed.

Reply:
@ritesh-singh I have solved all conflicts and removed unwanted strings. Just wanted to ask one thing whenever I rebase my forked repo, It gives so much of conflicts. How can I avoid them?

Reply:
Always work on the updated code, pull from upstream develop before starting and merge it with your local develop repo.

Before, creating PR, you can do the same to be avoid conflicts in PR>

Reply:
I always do rebase first, then it start giving conflicts on the files whose conflicts I have already resolved. So it makes me to resolve the same things again and again

Reply:
Should we use `git pull` or `git rebase`? Just asking since I read somewhere that `git rebase` is known to cause conflicts.

Reply:
I think `git rebase` is making conflicts every time, from now on I will do `git pull`

Reply:
http://thelazylog.com/git-rebase-or-git-pull/

Reply:
In your local repo, in develop branch. Do "git pull upstream develop", it will bring all the origin repo code in ur local repo.
The checkout to your working branch and merge with ur local develop.
"git merge develop " from your working branch.

Then, create a PR.